### PR TITLE
Add Nilotpal Pramanik to list of contestants.

### PR DESCRIPTION
**Pull request by goodday451999**

### CODE REVIEW

File: contestants.json [hunk 1]
```diff
@@ -223,23 +231,6 @@
...
-         "twitter":""
-	   
-      },
-      {
-		"name":"Shyam K Kotecha",
-		"imageurl":"contestant/pic.png",
-	        "about":"Newbie trying to learn opensource development.",
-         	"facebook":"",
-         	"github":"https://github.com/shyam49",
-         	"twitter":""
```
Comment (pradeepgangwar) [inline contestants.json]: @goodday451999 Why deleting other record?

Comment (goodday451999) [reply contestants.json]: nope

**Comment by pradeepgangwar** [comment]
Fix conflicts and do appropriate changes.

File: contestants.json [hunk 2]
```diff
@@ -247,6 +255,6 @@
          	"facebook":"",
          	"github":"https://github.com/shyam49",
          	"twitter":""
-      }
+      },
```
Comment (pradeepgangwar) [inline contestants.json]: No `,` here.

File: contestants.json [hunk 3]
```diff
@@ -120,6 +120,14 @@
          "github":"https://github.com/druvalcr28",
          "twitter":""
       },
+      {
+	       "name":"Nilotpal Pramanik",
+         "imageurl":"contestant/nilotpal.jpg",
```
Comment (pradeepgangwar) [inline contestants.json]: Add your image if referring to it here.

File: contestants.json [hunk 4]
```diff
@@ -255,6 +263,6 @@
 	   "facebook" : "https://facebook.com/utkarshsingh369",
 	   "github" : "https://github.com/utkarshsingh99",
 	   "twitter" : ""
-      }  
+      },
```
Comment (pradeepgangwar) [inline contestants.json]: no `,` here.

Comment (goodday451999) [reply contestants.json]: now there is no conflict

Comment (pradeepgangwar) [reply contestants.json]: But make requested changes. They are not ok.

**Comment by pradeepgangwar** [comment]
@goodday451999 Add image to patch-2 branch.

**Comment by pradeepgangwar** [comment]
I resolved your conflicts, next time please do it and squash your commits.

**Pull request merged**